### PR TITLE
Feat fixtures

### PR DESCRIPTION
**Pull request by dani-santos-code**
This is to implement dummy data so we can play with. Without that it's hard to implement search, indexes or filters. 

**To run all fixtures**, run this long command: `python manage.py loaddata department institution usertype learningresource mediatype module resourcemodule resourcetag tag userresource`

I also made improvements in our current Learning Resource Model. If you notice, when you name a column that has a foreign key with "id", then the "id" appears twice. For example, check your current learningresource.learningresource table. You see that "department_id_id"? Ugly! 😔 Solution is to just name the field "department" and django will name it "department_id" for you. That's pretty much everything! :) 

As you will see, I included a Module table and a relation between Modules and Learning Resources. I also made the module as well as the department indexes. 

We can talk more about indexes later! :)

### CODE REVIEW

File: learningresource/models.py [hunk 1]
```diff
@@ -18,11 +18,11 @@ class LearningResource(models.Model):
     title = models.CharField(max_length=255)
     url = models.TextField()
     description = models.TextField()
-    media_type_id = models.ForeignKey(MediaType, on_delete=models.CASCADE)
-    department_id = models.ForeignKey(Department, on_delete=models.CASCADE)
+    media_type = models.ForeignKey(MediaType, on_delete=models.CASCADE)
+    department = models.ForeignKey(Department, on_delete=models.CASCADE)
```
Comment (gabriela99) [inline learningresource/models.py]: Thanks for fixing it up. Looks good

**Review by gabriela99 — APPROVED** [review]
No need for changes from my part, works fine and code looks good.

File: learningresource/models.py (hunk 1 above)
Comment (faymarie) [reply learningresource/models.py]: Okay, I also removed the 'ids' within the accounts tables within my branch.

File: accounts/fixtures/department.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+  },
+  {
+    "model": "accounts.department",
+    "pk": 4,
+    "fields": {
+      "department_name": "Product Management"
+    }
+  }
+]
```
Comment (faymarie) [inline accounts/fixtures/department.json:37]: Here, I would add 'other' as an option. An example for other might be an English teacher or English learning books.

**Review by faymarie — APPROVED** [review]

**Pull request merged**